data/strings/strings.txt

```diff
@@ -4525,6 +4624,7 @@
     ja = Organic Maps で私の現在地が見れます！%1$@ または %2$@ オフラインマップはまだインストールしていないですか？ここからダウンロード： https://omaps.app/get
     ko = 현재 위치를 알아 보십시오. %1$@ 또는 %2$@ 열기
     lt = Labas, pažvelk į mano dabartinę vietą Organic Maps! %1$@ arba %2$@ Neturi atjungtinių žemėlapių? Atsisiųsk čia: https://omaps.app/get
+    lv = Apskatiet manu atrašanās vietu „Organic Maps“ lietotnē! %1$@ vai %2$@ Vai Jums nav nesaistes kartes? Lejupielādējiet to: https://omaps.app/get
```

Comment:
Here Github allows to conveniently review and proof-read any change.

Reply:
We need @ttrasuns' opinion on our proposed changes.